openapi/servicenow/client.bal

```diff
@@ -0,0 +1,477 @@
+        string  path = string `/api/sn_customerservice/case/field_values/${field_name}`;
+        map<anydata> queryParam = {"sysparm_dependent_value": sysparmDependentValue, "sysparm_limit": sysparmLimit, "sysparm_offset": sysparmOffset, "sysparm_reference_field_columns": sysparmReferenceFieldColumns, "sysparm_query": sysparmQuery, "sysparm_ref_qual_input": sysparmRefQualInput};
+        path = path + check getPathForQueryParam(queryParam);
+        map<any> headerValues = {"Accept": accept};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        json response = check self.clientEp-> get(path, accHeaders, targetType = json);
+        return response;
+    }
+    # Retrieves the list of possible field values for a choice or reference field for a specified record in the Case [sn_customerservice_case] table.
+    #
+    # + field_name - Name of a choice or reference field in the Case [sn_customerservice_case] table.
+    # + id - Sys_id of a record. Located in the Case [sn_customerservice_case] table to use when evaluating reference qualifiers.
+    # + sysparmDependentValue - Value to select in the Case [sn_customerservice_case] table choice field that the requested field is dependent on. Use only when requesting a choice field that is dependent on another field.
+    # + sysparmLimit - Maximum number of records to return. Unusually large sysparm_limit values can impact system performance. For requests that exceed this number of records, use the sysparm_offset parameter to paginate record retrieval.
+    # + sysparmOffset - Starting record index for which to begin retrieving records. Use this value to paginate record retrieval. This functionality enables the retrieval of all records, regardless of the number of records, in small manageable chunks.
+    # + sysparmReferenceFieldColumns - Comma-separated list of column names, from the table of the specified reference field, to return in the response.
+    # + sysparmQuery - Encoded query used to filter the result set. Queries defined using this parameter are appended to any encoded queries produced by reference qualifiers.
+    # + sysparmRefQualInput - Encoded set of field values representing a current object to pass to reference qualifiers that use JavaScript functions. Use only when requesting a reference field.
+    # + accept - Data format of the response body.
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function listFieldValuesByFieldInCaseById(string field_name, string id, string? sysparmDependentValue = (), decimal? sysparmLimit = (), decimal? sysparmOffset = (), string? sysparmReferenceFieldColumns = (), string? sysparmQuery = (), string? sysparmRefQualInput = (), string? accept = ()) returns json|error {
```

Comment:
The function name `listFieldValuesByFieldInCaseById` is confusing the user. I would say it should be `getValuesByCaseIdAndFieldName`.

Reply:
Fixed by https://github.com/kasthuriraajan/ballerinax-openapi-connectors/commit/7d722e1656dd4084768a6a6f0aab1753b97a5d98